Documentation/contributing.rst

```diff
@@ -1764,6 +1785,8 @@ If you intent to release a new minor release, see the
 
       This step requires you to login with ``docker login`` first and it will
       require your Docker hub ID to have access to the ``Cilium`` organization.
+      You can also trigger a build on DockerHub directly if you have adequate
```

Comment:
s/also/alternatively ? I don't think there's any point in building locally AND running the build via DockerHub.

Reply:
Fixed in latest upload.